pkgs/tools/text/base16384/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "base16384";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "fumiama";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-5Dfu88aAVtpNAwHAo+LOxe+jpgDhNdHYkWex42dCK0s=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with lib; {
+    description = "Encode binary files to printable utf16be";
+    homepage = "https://github.com/fumiama/base16384";
+    changelog = "https://github.com/fumiama/base16384/releases/tag/v${version}";
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```

https://github.com/fumiama/base16384/commit/e270980b37cc23d6f69dc2a4c4feb0f676334369